guides/demo.md

```diff
@@ -0,0 +1,124 @@
+# Wanda Demo
+
+## Overview
+
+Wanda Demo showcases the capabilities of the Trento project. It offers a simulated environment where we can demonstrate the seamless interaction between Web and Wanda. The configuration file provided in the demo simulates the fact gathering process, allowing us to perform a comprehensive evaluation of checks and displaying these on the web
+
+## How to setup Wanda?
+
+To set up a local development environment for hacking on Wanda, you need to follow the instructions provided in [hack on wanda](/guides/development/hack_on_wanda.md).
+
+## How to run Wanda in Demo mode?
+
+Instead of starting the local developer environment we need to start wanda in demo mode we can use this command
+
+```
+DATABASE_URL=ecto://postgres:postgres@localhost:5434/wanda_dev AMQP_URL=amqp://wanda:wanda@localhost:5674 SECRET_KEY_BASE=Tbp26GilFTZOXafb7FNVqt4dFQdeb7FAJ++am7ItYx2sMzYaPSB9SwUczdJu6AhQ CORS_ORIGIN=http://localhost:4000 ACCESS_TOKEN_ENC_SECRET=s2ZdE+3+ke1USHEJ5O45KT364KiXPYaB9cJPdH3p60t8yT0nkLexLBNw8TFSzC7k  PORT=4001  MIX_ENV=demo iex -S mix phx.server
+```
+
+Explanation:
+
+- DATABASE_URL: The URL to connect to the Postgres database running in the Docker container.
+- AMQP_URL: The URL for RabbitMQ, used for message exchange.
+- SECRET_KEY_BASE: The secret key for Wanda, keep it secure.
+- CORS_ORIGIN: The origin URL from where API requests are allowed (pointing to web).
+- ACCESS_TOKEN_ENC_SECRET: Secret key for encrypting access tokens.
+- PORT: The port on which Wanda will run (4001 in this case).
+- MIX_ENV: The environment in which Wanda will run (set to "demo" for the demo environment).
+
+## Modify the demo facts configuration
```

Comment:
I would probably focus on the structure of the yaml and its meaning first, cause then it would be easier to grasp how to add things

```yaml
targets:
  <target_reference>: <target_id>

facts:
  <check_id>:
    <fact_name>:
      <target_reference>: <Fact Value>
    <another_fact_name>:
      <another_target_reference>: <Fact Value>
```

where the `targets` section allows to define handles for targets' uuids so that we do not have to repeat uuids all around and keep the yaml less noisy.

while the `facts` section allows to define which `<Fact Value>` a target (referenced by `<target_reference>`) should return for a specific fact (`<fact_name>`) on a given check (`<check_id>`).

Then we can say that we can add as many targets as needed and as many checks/facts as needed.